TempleEight/docs/guide/regeneration.md

```diff
@@ -3,3 +3,84 @@ id: regeneration
 title: Regeneration
 sidebar_label: Regeneration
 ---
+
+In addition to generating code from a Templefile a single time to being project development, the Temple CLI also allows for regeneration of code.
```

Comment:
This first part doesn't make sense?

Reply:
Lol I wrote that while also trying to focus on Dav-andy